addons/project/models/project.py

```diff
@@ -1171,6 +1172,11 @@ def _compute_is_private(self):
         for task in self:
             task.is_private = not task.project_id and not task.parent_id
 
+    @api.depends('parent_id', 'parent_id.ancestor_id')
```

Comment:
This is redundant, you can only specify `parent_id.ancestor_id` and it will do the same.
```suggestion
    @api.depends('parent_id.ancestor_id')
```

Reply:
Done. :slightly_smiling_face: